src/Native/Runtime/inc/eetype.h

```diff
@@ -444,6 +444,11 @@ class EEType
     bool IsGeneric()
         { return (m_usFlags & IsGenericFlag) != 0; }
 
+#if defined(CORERT)
+    ModuleManager* GetModule()
```

Comment:
(Or maybe ReadyToRunModule so that it doesn't clash with the Module that is already defined in Redhawk and corresponds to the OS module.)